### ISSUE TITLE
Bug - Controller - IKEA E2001/E2002 STYRBAR Remote control

### ISSUE THREAD

**Issue by lilium360** [EPMatt/awesome-ha-blueprints#386]
### Blueprint name

Controller - IKEA E2001/E2002 STYRBAR Remote control

### Home Assistant Core Version

2022.8.1

### Home Assistant Installation Type

Home Assistant Core

### Description

When you press the UP button for turn the lights on, lights turn on but also all the Aqara smartplug connected in the Zigbee Ecosystem. Same problem when you press the DOWN button.

### Automation YAML config

```yaml
alias: Interruttore comodini - comando
description: ""
use_blueprint:
  path: EPMatt/ikea_e2001_e2002.yaml
  input:
    integration: ZHA
    controller_device: eb64164f6fd77a709a0b4a0d5c51e299
    helper_last_controller_event: input_text.interruttore_camera_da_letto_azioni
    action_button_left_short: []
    action_button_up_short:
      - service: light.turn_on
        data:
          transition: 3
          color_temp: 500
          brightness: 1
        target:
          entity_id: light.comodini
    action_button_down_short:
      - service: light.turn_off
        data:
          transition: 3
        target:
          entity_id: light.comodini
    action_button_down_long:
      - parallel:
          - device_id: e7c3c5d32a140c0cd597b419a8f8cbbe
            domain: light
            entity_id: light.comodino_ilyo
            type: brightness_decrease
          - device_id: 80bc2eec30f18281662d4fdf57d67ba9
            domain: light
            entity_id: light.comodino_destra
            type: brightness_decrease
    action_button_up_long:
      - parallel:
          - device_id: e7c3c5d32a140c0cd597b419a8f8cbbe
            domain: light
            entity_id: light.comodino_ilyo
            type: brightness_increase
          - device_id: 80bc2eec30f18281662d4fdf57d67ba9
            domain: light
            entity_id: light.comodino_destra
            type: brightness_increase
    button_up_long_loop: true
    button_down_long_loop: true
    button_up_long_max_loop_repeats: 100
    button_down_long_max_loop_repeats: 100
```

### To Reproduce

1. Go to 'remote location in your house'
2. Click on 'up button on the remote'
3. Trigger the automation 'turn the lights on'

### Expected behavior

When you press the UP button for turn the lights on, lights turn on but also all the Aqara smartplug connected in the Zigbee Ecosystem. Same problem when you press the DOWN button.

### Actual Behaviour

same of Expected behavior

### Additional Details

- [ ] I'd like to help developing a fix for this issue.

**Comment by ioExpander** [comment]
Hi. I had a similar issue with Ikea remotes and Aquara/xiaomi plugs. I do not think this is related to this automation but rather the quirky way Aquara plugs are implemented. There are several forum posts documenting this issue. On my side I've replaced the Aquara plugs to resolve the issue.

**Comment by Egglestron** [comment]
Best thread documenting the issue so far: https://github.com/Koenkk/zigbee2mqtt/issues/13903#issuecomment-1244720072.

tl;dr: you can try downgrading your devices' firmware.

**Comment by EPMatt** [comment]
Hey @lilium360,

great news - after a long hiatus, [we are back](https://github.com/EPMatt/awesome-ha-blueprints/discussions/746)! 🚀

I know it must have been frustrating to see this project slow down, and I can’t thank you all enough for keeping it alive while I was away. Looking at the time, effort, and dedication you've put in truly leaves me speechless. This is exactly what motivates me to jump back in and bring my energy back to the project.

As documented by @Egglestron and @ioExpander  above, this looks like a problem related to Aqara / Xiaomi smart plugs rather than this blueprint itself. Therefore, I'm closing this issue as not planned.

However, feel free to open a new issue in case you are experiencing other problems with one of our blueprints. Looking forward to see you around the Awesome HA Blueprints community in the future! 🚀
